cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -146,6 +146,17 @@ def _validate(self, instance_type: str, image: str):
                     ),
                     FailureLevel.ERROR,
                 )
+        unsupported = ["p3", "p2", "g3", "g2"]
```

Comment:
let's create a constant in the constants.py file, called for example: `NVIDIA_OPENRM_UNSUPPORTED_INSTANCE_TYPES`